thorlcr/activities/diskread/thdiskreadslave.cpp

```diff
@@ -982,7 +1010,8 @@ class CDiskGroupAggregateSlave
 // IRowStream
     virtual void stop()
     {
-        partHandler.clear();
```

Comment:
Jake.  I think this clear would mean it would crash if it was used inside a child query.